_posts/2020-12-24-JabRef5-2.md

```diff
@@ -0,0 +1,35 @@
+---
+title:  🎄 JabRef 5.2 Release 🎄
+id: jabref-5-2
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)"
+---
+
+We are proud to announce the release of the next milestone of JabRef. Today we are releasing JabRef 5.2. Just in time for the holidays!
+We are also happy to announce that we have a new developer, [Domninik Voigt](https://github.com/DominikVoigt), who implemented a proof of concept tool for systematic literature search in JabRef as part of his thesis. The feature still needs some work until it's usable for non-technical users.
```

Comment:
who worked on JabRef as part of his master? thesis and now joined the ranks of core developers.